arm-network/2017-06-01/swagger/loadBalancer.json

```diff
@@ -557,10 +558,11 @@
         },
         "protocol": {
           "type": "string",
-          "description": "The transport protocol for the endpoint. Possible values are: 'Udp' or 'Tcp'",
+          "description": "The transport protocol for the endpoint. Possible values are: 'Udp,' 'Tcp,' or 'All.'",
```

Comment:
Selecting "all" is part of how the customer makes use of ILB HA. This setting is used in conjunction with port 0 on the frontend and backend ports.

Yes, All will still be valid if there is a new transport protocol. The idea is that all traffic is allowed in, and then our service acts as a firewall, only allowing certain traffic through to the customer VMs.